homeassistant/components/template/sensor.py

```diff
@@ -132,7 +129,7 @@ def validate_last_reset(val):
 
 def extra_validation_checks(val):
     """Run extra validation checks."""
-    if CONF_TRIGGER in val:
+    if CONF_TRIGGERS in val or CONF_TRIGGER in val:
```

Comment:
As I understand it, the configuration is rewritten and there can only be `triggers`, not `trigger` after schema validation. So why do the check both?